src/e-cherry-pick.js

```diff
@@ -131,143 +138,100 @@ program
-            },
-          ],
+        fs.writeFileSync(`${electronPath}/${patchPath}/${patchName}`, patch);
+        fs.writeFileSync(`${electronPath}/${patchPath}/.patches`, newPatchList);
+
+        d(`Committing changes`);
+        const commitMsg = `chore: cherry-pick ${shortCommit} from ${patchDirName}`;
+        cp.execSync(`git add ${patchPath}`, { cwd: electronPath });
+        cp.execSync(`git commit -m "${commitMsg}"`, {
+          cwd: electronPath,
+          stdio: 'ignore',
         });
 
-        d(`creating commit tree=${tree.sha} parent=${targetSha}`);
-        const { data: commit } = await octokit.git.createCommit({
-          owner: 'electron',
-          repo: 'electron',
-          tree: tree.sha,
-          parents: [targetSha],
-          message: `chore: cherry-pick ${shortCommit} from ${patchDirName}`,
+        // Push the changes to the remote.
+        cp.execSync(`git push origin ${branchName}`, {
+          cwd: electronPath,
+          stdio: 'ignore',
         });
 
-        d(`creating ref`);
-        await octokit.git.createRef({
-          owner: 'electron',
-          repo: 'electron',
-          ref: `refs/heads/${branchName}`,
-          sha: commit.sha,
-        });
+        const comparePath = `${target}...${pullRequestSource(branchName)}`;
+        const queryParams = { expand: 1 };
 
-        d(`creating pr`);
-        const { data: pr } = await octokit.pulls.create({
-          owner: 'electron',
-          repo: 'electron',
-          head: `electron:${branchName}`,
-          base: target,
-          title: `chore: cherry-pick ${shortCommit} from ${patchDirName}`,
-          body: `${commitMessage}\n\nNotes: ${
+        queryParams.title = commitMsg;
+        (queryParams.labels = `target,backport-check-skip,semver/patch,${
+          security ? 'security 🔒' : ''
+        }`),
+          (queryParams.body = `${commitMessage}\n\nNotes: ${
             bugNumber
               ? security
                 ? `Security: backported fix for ${cve || bugNumber}.`
                 : `Backported fix for ${bugNumber}.`
               : `<!-- couldn't find bug number -->`
-          }`,
-          maintainer_can_modify: true,
-        });
+          }`);
 
-        d(`labelling pr`);
-        await octokit.issues.update({
-          owner: 'electron',
-          repo: 'electron',
-          issue_number: pr.number,
-          labels: [
-            target,
-            'backport-check-skip',
-            'semver/patch',
-            ...(security ? ['security 🔒'] : []),
-          ],
-        });
+        open(
+          `${remotes.electron.origin}/compare/${comparePath}?${querystring.stringify(queryParams)}`,
+        );
+        console.log(`${color.success} Created cherry-pick PR to ${target}`);
 
-        console.log(`Created cherry-pick PR to ${target}: ${pr.html_url}`);
+        d(`Removing temporary electron directory at ${tmp}`);
+        fs.rmSync(tmp, { recursive: true, force: true });
```

Comment:
Doing this inside the loop will break multi backport because we clone outside the loop but delete inside. Let's just replace this with a spawn of `git clean -fdx` instead to ensure the checkout is clean between runs of the loop